internal/parse/parser.go

```diff
@@ -177,15 +177,18 @@ func (p *Parser) skipByteFind(b byte) bool {
 	return false
 }
 
-// skipSpaces advances the parser jumping over consecutive spaces. It stops when
-// finding a non-space character. Returns true if the parser position was
-// actually changed, false otherwise.
-func (p *Parser) skipSpaces() bool {
+// skipBlanks advances the parser jumping over consecutive spaces, tabs or new
+// lines. It stops when finding a non-blank character. Returns true if the
+// parser position was actually changed, false otherwise.
+func (p *Parser) skipBlanks() bool {
```

Comment:
The middle sentence seems redundant to me. Perhaps something like this?
```go
// skipBlanks advances the parser past spaces, tabs and newlines. Returns whether
// the parser position was changed.
```

Reply:
Simpler and clearer. Thanks!